python/cudf/cudf/pandas/__init__.py

```diff
@@ -26,6 +28,27 @@
 }
 
 
+def _ensure_managed_memory_is_supported(rmm_mode):
```

Comment:
Can you move this function into pylibcudf so that we can re-use it for cudf-polars as well?

Reply:
Sure! Done in 12f0973.